### PR TITLE
Update patch deps

### PR DESCRIPTION
**Pull request by matthewdavidson**
See comments inline...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -97,7 +97,7 @@
     "danger": "^3.8.6",
     "del": "^3.0.0",
     "enzyme": "^3.7.0",
-    "enzyme-adapter-react-16": "^1.6.0",
+    "enzyme-adapter-react-16": "^1.7.0",
```
Comment (matthewdavidson) [inline package.json]: I cannot find a changelog entry for this, but the tests still pass so 🤷‍♂️

File: package.json [hunk 2]
```diff
@@ -118,22 +118,22 @@
     "gulp-rename": "^1.4.0",
     "gulp-svgmin": "^1.2.3",
     "gulp-theo": "^2.0.0",
-    "husky": "^1.1.2",
+    "husky": "^1.2.0",
```
Comment (matthewdavidson) [inline package.json]: I tested this by: 
- clearing `node_modules`
- clearing git hooks
- npm install
- add eslint error to javascript file
- observe husky doing it's thing

File: package.json [hunk 3]
```diff
@@ -118,22 +118,22 @@
...
     "jest": "^21.2.1",
     "js-yaml": "^3.12.0",
     "json-loader": "^0.5.4",
     "lerna": "^3.4.3",
-    "lint-staged": "^8.0.4",
+    "lint-staged": "^8.0.5",
```
Comment (matthewdavidson) [inline package.json]: My test for husky above covered this too.

File: package.json [hunk 4]
```diff
@@ -118,22 +118,22 @@
...
     "lodash": "^4.17.11",
-    "marked": "^0.5.1",
+    "marked": "^0.5.2",
     "merge-stream": "^1.0.1",
-    "node-sass": "^4.9.3",
+    "node-sass": "^4.10.0",
```
Comment (matthewdavidson) [inline package.json]: The noisy deprecation warning for importing css is gone now. See https://github.com/sass/node-sass/releases/tag/v4.11.0.

File: package.json [hunk 5]
```diff
@@ -143,11 +143,11 @@
     "react-test-renderer": "16.4.2",
     "rimraf": "^2.5.4",
     "sass-loader": "^7.1.0",
-    "sassdoc": "^2.5.0",
+    "sassdoc": "^2.5.1",
     "semver": "^5.6.0",
     "static-site-generator-webpack-plugin": "github:skyscanner/static-site-generator-webpack-plugin#7bac616",
     "style-loader": "^0.22.1",
-    "stylelint": "^9.6.0",
+    "stylelint": "^9.8.0",
```
Comment (matthewdavidson) [inline package.json]: Lint still passes ...

File: package.json [hunk 6]
```diff
@@ -191,6 +191,10 @@
       "json-loader",
       "postcss-loader",
       "raw-loader",
+      "react",
+      "react-dom",
+      "react-native",
+      "react-test-renderer",
```
Comment (matthewdavidson) [inline package.json]: I added react et al to the greenkeeper ignore list as _we_ decide when we upgrade these deps.

**Comment by shaundon** [comment]
If CI is happy, I'm happy. None of this will affect `npm start`, `npm run native` or `npm docs` or other stuff we need that isn't part of CI, so should be fine.

**Comment by matthewdavidson** [comment]
@shaundon actually `marked`, `node-sass` and `sassdoc` will affect `npm start` / `npm run docs` ... I have checked em though so 👍

**Comment by matthewdavidson** [comment]
@shaundon approve pleeez 🙏

**Pull request merged**